src/library/trackset/setlogfeature.cpp

```diff
@@ -136,21 +141,36 @@ void SetlogFeature::onRightClickChild(const QPoint& globalPos, QModelIndex index
 }
 
 QList<BasePlaylistFeature::IdAndLabel> SetlogFeature::createPlaylistLabels() {
-    QList<BasePlaylistFeature::IdAndLabel> playlistLabels;
+    return QList<BasePlaylistFeature::IdAndLabel>();
```

Comment:
can we dispose the function now?

Reply:
The prototype requires it to be overridden. If I change the prototype, other derived classes may miss implementation when first implemented.

Reply:
Due to your changes the pure virtual implementation + overriding the 
QModelIndex BasePlaylistFeature::constructChildModel()
Is not longer reasonable. Can you check if it's possible to move both to PlaylistFeature?